test/mocks/mockUsers.js

```diff
@@ -0,0 +1,31 @@
+exports.mockUser = {
+  firstName: 'ricardo',
+  lastName: 'moreno',
+  email: 'ricardo.a.moreno@wolox.co',
```

Comment:
Remove any real people information from a public repository (or even a private one)

Reply:
I replaced it with generic user information